src/Kernel.RedisSupport/Extensions/RedisExtension.cs

```diff
@@ -18,7 +18,17 @@ public static string GetRedisCacheHashCode(this IEnumerable<Guid> guids, params
 
         foreach (var arg in additionalArguments)
         {
-          cache += arg.GetHashCode();
+          if (arg is string)
+          {
+            foreach (char value in arg.ToString().Replace(" ", ""))
+            {
+              cache += cache * 17 + value.GetHashCode();
```

Comment:
maybe multiply a value hash? when you multiply cache it creates a dependence on additional arguments' order, so if in consumer you set key with additional args like (string, other arg) and in request like (other arg, your string) - they won't be same keys

Reply:
and i think that you can keep spaces in the string, they have their own hash